spectrocloud/cluster_common_host.go

```diff
@@ -99,19 +99,15 @@ func flattenHostConfig(hostConfig *models.V1HostClusterConfig) []interface{} {
 
 func flattenSourceRanges(hostConfig *models.V1HostClusterConfig) string {
 	sourceRanges := make([]string, 0)
-	for _, r := range hostConfig.ClusterEndpoint.Config.LoadBalancerConfig.LoadBalancerSourceRanges {
-		sourceRanges = append(sourceRanges, r)
-	}
+	copy(sourceRanges, hostConfig.ClusterEndpoint.Config.LoadBalancerConfig.LoadBalancerSourceRanges)
 	return strings.Join(sourceRanges, ",")
 }
 
-func flattenExternalIPs(hostConfig *models.V1HostClusterConfig) string {
-	externalIPs := make([]string, 0)
-	for _, ip := range hostConfig.ClusterEndpoint.Config.LoadBalancerConfig.ExternalIPs {
-		externalIPs = append(externalIPs, ip)
-	}
-	return strings.Join(externalIPs, ",")
-}
+// func flattenExternalIPs(hostConfig *models.V1HostClusterConfig) string {
```

Comment:
Frankly I cannot recall what was the usage, I think we can delete it was probably for virtual clusters.